drivers/net/ethernet/renesas/rswitch_tc_u32.c

```diff
@@ -48,14 +73,274 @@ static void rswitch_init_u32_redirect_action(struct rswitch_tc_filter *cfg,
 	}
 }
 
+static int compare_pf_entries(const void *lhs, const void *rhs)
+{
+	struct rswitch_pf_entry *lhs_entry = (struct rswitch_pf_entry *)(lhs);
```

Comment:
I think we should save const qualifier for lhs/rhs entries and remove (const void *) cast

Reply:
This function was removed as it is not needed anymore, so the issue is fixed.